vignettes/extending-jmpost.Rmd

```diff
@@ -232,3 +232,188 @@ parameter { real mu;      // non-whitespace after opening `{`
   x ~ normal(mu, sigma);
 } // some comment         // non-whitespace after closing `}`
 ```
+
+## Stan Data Objects
+
+When writing your own Stan code to extend jmpost it is important to note that
```

Comment:
Maybe consistently put jmpost in backticks (code formatting) or something like that?